lib/OpenQA/Script/CloneJob.pm

```diff
@@ -205,39 +205,57 @@ sub openqa_baseurl {
     return $local_url->scheme . '://' . $local_url->host . $port;
 }
 
+sub get_deps {
+    my ($job, $options, $job_type) = @_;
+
+    my ($chained, $directly_chained, $parallel);
+    unless ($options->{'skip-deps'}) {
+        unless ($options->{'skip-chained-deps'}) {
+            $chained          = $job->{$job_type}->{Chained};
+            $directly_chained = $job->{$job_type}->{'Directly chained'};
+        }
+        $parallel = $job->{$job_type}->{Parallel};
+    }
+    $chained          //= [];
+    $directly_chained //= [];
+    $parallel         //= [];
+
+    return ($chained, $directly_chained, $parallel);
```

Comment:
```suggestion
    return ($chained // [], $directly_chained // [], $parallel // []);
```

Reply:
I simply copy/paste without trying to improve, thx for getting this!